erpnext/shopping_cart/cart.py

```diff
@@ -334,23 +334,28 @@ def set_price_list_and_rate(quotation, cart_settings):
 		# set it in cookies for using in product page
 		frappe.local.cookie_manager.set_cookie("selling_price_list", quotation.selling_price_list)
 
-def _set_price_list(quotation, cart_settings):
+def _set_price_list(cart_settings, quotation={}):
```

Comment:
Using mutable values as default arguments is discouraged

* https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/
* https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments
* https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html